.github/workflows/quarkus-snapshot.yaml

```diff
@@ -16,7 +16,7 @@ env:
   # Repo specific setting #
   #########################
 
-  ECOSYSTEM_CI_REPO_PATH: quarkus-renarde
+  ECOSYSTEM_CI_REPO_PATH: quarkiverse-renarde
```

Comment:
Per convention, quarkiverse extensions start with `quarkiverse-` in Ecosystem CI.